### DISCUSSION TITLE
This Process designer is built with Blazor.Diagrams

### DISCUSSION THREAD

**Discussion by ATEEKGIT** [Blazor-Diagrams/Blazor.Diagrams#426] [Show and tell]
Here is the video of live application which uses Blazor-Diagrams.

[NileDesk Process Designer ](https://www.youtube.com/watch?v=p61b8sXK52w)

Thank you Haytam :)

**Comment by zHaytam** [comment]
Oh my god, this is so insane! Thank you for using this library ❤️
